tests/operators/fermi_hubbard_test.py

```diff
@@ -0,0 +1,130 @@
+# (C) Copyright IBM 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Tests for Fermi-Hubbard operator."""
+
+from __future__ import annotations
+
+from math import comb
+
+import numpy as np
+import scipy
+
+import ffsim
+from ffsim.operators.fermi_hubbard import fermi_hubbard
+
+
+def test_non_interacting_FH_operator():
+    """Test non-interacting Fermi-Hubbard operator."""
+
+    # open boundary conditions
+    op = fermi_hubbard(norb=4, t_hop=1)
+    ham = ffsim.linear_operator(op, norb=4, nelec=(2, 2))
+    eigs, _ = scipy.sparse.linalg.eigsh(ham, which="SA", k=1)
+    E = -4.472135955000
```

Comment:
Variable names should be all lowercase and descriptive (i.e. `energy` instead of `e`. In this case, I would avoid defining a variable altogether and just put the number directly in the assert statement.